apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1.IntegrationTests/Google.Cloud.Datastore.V1.IntegrationTests.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Google.Api.Gax.Grpc.Testing" Version="[4.4.0, 5.0.0)" />
+    <PackageReference Include="Google.Cloud.Firestore.Admin.V1" Version="3.3.0" />
```

Comment:
It looks like you've added this manually - which means it'll be overwritten next time we regenerate the projects. Please edit apis/apis.json to add the test reference - there should be examples you can follow there.

Reply:
Done. I've added the dependency in the apis.json file and executed `./generateapis.sh Google.Cloud.Datastore.V1`
 with it.